### DISCUSSION TITLE
Help with table extraction settings when not all columns in table have a vertical line

### DISCUSSION THREAD

**Discussion by sfc-gh-svadakath** [jsvine/pdfplumber#1134] [Ask for help with specific PDFs]
Hi, I'd like to table extract  five columns from Schedule A (pages 5-36) in attached PDF. The challenges is that it only picks the middle three columns due to the left and right most columns not having enclosing vertical line. See screenshot. Is there a way using table settings to extract all five columns for just Schedule A table using pdfplumber? Appreciate any pointers. Thank you

<img width="926" alt="image" src="https://github.com/jsvine/pdfplumber/assets/122318553/48e788c2-25a9-48af-a5db-b077062ba5c7">

Here's my code snippet looking at just page 5. 

`import pdfplumber
import pandas as pd

# Set display options to show all rows and columns
pd.set_option('display.max_rows', None)  # None means unlimited
pd.set_option('display.max_columns', None)  # None means unlimited

# Specify the path to your PDF file
pdf_file_path = "/Users/svadakath/Data/AlertPDF/RBC.pdf"

pdf = pdfplumber.open(pdf_file_path)
table=pdf.pages[5].extract_table()
pd.DataFrame(table[1::],columns=table[1])

table_settings = {
    "vertical_strategy": "lines",
    "horizontal_strategy": "lines"
}
pdf = pdfplumber.open(pdf_file_path)
table=pdf.pages[5].extract_table(table_settings)
pd.DataFrame(table[1::],columns=table[0])` 
  

[RBC.pdf](https://github.com/jsvine/pdfplumber/files/15171849/RBC.pdf)

**Comment by jsvine** [comment]
Hi @sfc-gh-svadakath, and thanks for your interest in `pdfplumber`. The solution proposed in this comment may work for your situation, too: https://github.com/jsvine/pdfplumber/discussions/617#discussioncomment-2284539

Let us know if not.
